src/components/Layout/HomeContent.js

```diff
@@ -131,100 +131,100 @@ export function HomeContent() {
             React
           </h1>
           <p className="text-4xl font-display max-w-lg md:max-w-full py-1 text-center text-secondary dark:text-primary-dark leading-snug self-center">
-            The library for web and native user interfaces
+            A biblioteca para web e interfaces de usuário nativas
           </p>
           <div className="mt-5 self-center flex gap-2 w-full sm:w-auto flex-col sm:flex-row">
             <ButtonLink
               href={'/learn'}
               type="primary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="Learn React">
-              Learn React
+              label="Aprenda React">
+              Aprenda React
             </ButtonLink>
             <ButtonLink
               href={'/reference/react'}
               type="secondary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="API Reference">
-              API Reference
+              label="API Referência">
+              API Referência
             </ButtonLink>
           </div>
         </div>
 
         <Section background="left-card">
           <Center>
-            <Header>Create user interfaces from components</Header>
+            <Header>Crie interfaces de usuário a partir de componentes</Header>
             <Para>
-              React lets you build user interfaces out of individual pieces
-              called components. Create your own React components like{' '}
-              <Code>Thumbnail</Code>, <Code>LikeButton</Code>, and{' '}
-              <Code>Video</Code>. Then combine them into entire screens, pages,
-              and apps.
+              O React permite que você crie interfaces de usuário a partir de
+              peças individuais chamadas de componentes. Crie seus próprios
+              componentes React como <Code>Thumbnail</Code>,{' '}
+              <Code>LikeButton</Code>, e <Code>Video</Code>. Em seguida,
+              combine-os em telas inteiras, páginas, e aplicativos.
             </Para>
           </Center>
           <FullBleed>
             <Example1 />
           </FullBleed>
           <Center>
             <Para>
-              Whether you work on your own or with thousands of other
-              developers, using React feels the same. It is designed to let you
-              seamlessly combine components written by independent people,
-              teams, and organizations.
+              Quer você trabalhe sozinho ou com milhares de outros
+              desenvolvedores, usar o React parece o mesmo. Ele foi projetado
```

Comment:
```suggestion
              desenvolvedores, é possível usar o React da mesma maneira. Ele foi projetado
```

O texto assim fica mais longo, mas o termo `feels the same` é quase uma gíria sem um equivalente em português. Acho que o ideal é reformular para uma linguagem mais amigável para o português.